lib/std/os/windows/user32.zig

```diff
@@ -113,16 +245,802 @@ pub const WM_RBUTTONDBLCLK = 0x0206;
 pub const WM_MBUTTONDOWN = 0x0207;
 pub const WM_MBUTTONUP = 0x0208;
 pub const WM_MBUTTONDBLCLK = 0x0209;
+pub const WM_MOUSELAST = 0x0209;
```

Comment:
We shouldn't include `*LAST` and `*FIRST` constants IMO

Reply:
They can still be used fine with for example `GetMessageW` with the min and max filter arguments provided as `xxx_FIRST` to `xxx_LAST`

Reply:
The *LAST ones move on most updates; is it really useful to have the wrong values?

e.g. this PR changes `WM_MOUSELAST` from `0x020A` to `0x0209`, however in windows its now up to `0x020E`

Reply:
I would say that you can't handle what you didn't know existed at the time, but fair point. I've removed those constants.